twitter_ads/http.py

```diff
@@ -141,6 +141,7 @@ def __init__(self, code, headers, **kwargs):
             raw_response_body = self._raw_body
 
         try:
+            raw_response_body = str(raw_response_body)
```

Comment:
That was how I found the code, so I didn't change that. I did just change location of the stringify just now though.